features-json/tls1-1.json

```
@@ -317,7 +333,7 @@
       "7.12":"y"
     }
   },
-  "notes":"",
+  "notes":"TLS 1.0 & 1.2 are deprecated in Chrome, Edge, Firefox, Internet Explorer 11, & Safari. Support will be removed in March 2020.",
```

It should be "TLS 1.0 & 1.1", shouldn't it? 0:-)

```suggestion
  "notes":"TLS 1.0 & 1.1 are deprecated in Chrome, Edge, Firefox, Internet Explorer 11, & Safari. Support will be removed in March 2020.",
```